modules/ROOT/pages/file/file-list.adoc

```diff
@@ -3,35 +3,83 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The List operation returns a List of Messages, where each message represents any file or folder found within the Directory Path (`directoryPath`).
 
-By default, the operation does not read or list files or folders within any sub-folders of `directoryPath`.
 
-To list files or folders within any sub-folders, you can set the `recursive` parameter to `true`.
+The List operation returns an Array of Messages, where each message represents any file
+or folder (and their attributes) found in the path pointed by the `directoryPath` parameter.
```

Comment:
doesn't represent, it holds in the payload of the message the content of the file and in the attributes section the metadata of the element